src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -605,6 +605,15 @@
 	IEmoji getEmojiByName(String name);
 
 	/**
+	 * This adds a custom emoji to the guild.
+	 *
+	 * @param name The name, <b>without colons</b>.
+	 * @param image The BufferedImage of the Emoji.
+	 * @return The emoji.
```

Comment:
- This takes `Image` not `BufferedImage`
- This should mention it is only available to whitelisted bots to avoid confusion
- `@param name` should mention it only takes alphanumeric input between 2-32 chars
- Maybe a `@throws` documentation for `DiscordException`? Not sure if that's needed here.